Dockerfile

```diff
@@ -58,8 +62,8 @@ RUN apt-get update && \
     freetds-dev \
     libsasl2-dev \
     unzip \
-    libsasl2-modules-gssapi-mit && \
-    apt-get clean && \
+    libsasl2-modules-gssapi-mit
+RUN apt-get clean && \
```

Comment:
we can't separate the cleaning from the updating and installing layer: it make the final image bigger.

https://betterprogramming.pub/how-to-improve-docker-image-size-with-layers-3ad62be0da9b